src/Mbstring/Mbstring.php

```diff
@@ -871,6 +871,49 @@ public static function mb_str_pad(string $string, int $length, string $pad_strin
         }
     }
 
+    public static function mb_ucfirst(string $string, ?string $encoding = null): string {
```

Comment:
It is good to add line breaks according to [Symfony Coding Standards](https://symfony.com/doc/current/contributing/code/standards.html).

```suggestion
    public static function mb_ucfirst(string $string, ?string $encoding = null): string
    {
```

Reply:
It seems good to add `mb_ucfirst` and `mb_ucfirst` to Implemented in the class header.

https://github.com/symfony/polyfill/pull/471/files#diff-3e84767d56f3132c8784f8423c914322116cef95a35b583c80070f666e9360d7R51